tests/test_e2e.py

```diff
@@ -193,6 +193,7 @@ def _build_app(cls,
                 http_client=http_client or MinimalHttpClient(),
                 enable_broker_on_windows=_PYMSALRUNTIME_INSTALLED,
                 enable_broker_on_mac=_PYMSALRUNTIME_INSTALLED,
+                enable_broker_on_linux=_PYMSALRUNTIME_INSTALLED,
```

Comment:
Shall we also add this?
```suggestion
                enable_broker_on_linux=_PYMSALRUNTIME_INSTALLED,
                enable_broker_on_wsl=_PYMSALRUNTIME_INSTALLED,
```